bridge/e2e/src/user-menu.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import { AppPage } from './app.po';
+import { browser, by, element, logging, protractor } from 'protractor';
+
+import { takeScreenshot } from './utils';
+
+const {execSync} = require('child_process');
+
+// run with: ng e2e --dev-server-target= --base-url=http://localhost:3000/
+describe('User Menu', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should open the user menu and copy api token', async () => {
+    await page.navigateTo();
+
+    await element(
+      by.xpath('//*[@uitestid="keptn-nav-userMenu"]')
+    ).click();
+    takeScreenshot('user-menu-open.png');
+
+    await element(
+      by.xpath('//*[@uitestid="keptn-nav-copyKeptnApiToken"]/ktb-copy-to-clipboard/div/div[2]/dt-copy-to-clipboard/button')
+    ).click();
+
+    await browser.executeScript(() => {
+      let el = document.createElement('input');
+      el.setAttribute('id', 'apiTokenTempInput');
+      document.getElementsByTagName('body')[0].appendChild(el);
+    });
+
+    const apiTokenTempInput = element(by.id('apiTokenTempInput'));
+    await apiTokenTempInput.sendKeys(protractor.Key.chord(protractor.Key.CONTROL, 'v'));
+
+    const val = await apiTokenTempInput.getAttribute('value');
+    const apiToken = Buffer.from(execSync('kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token}').toString(), 'base64').toString();
```

Comment:
Once my PR is merged, this should not be needed. I am providing `KEPTN_API_TOKEN` as an environment variable.